composer.json

```diff
@@ -39,14 +39,16 @@
         "phpdocumentor/reflection-docblock": "^3.0",
         "symfony/cache": "^3.1",
         "symfony/dependency-injection": "^2.7 || ^3.0",
-        "symfony/framework-bundle": "^3.1",
         "symfony/doctrine-bridge": "^2.8 || ^3.0",
         "symfony/phpunit-bridge": "^2.7 || ^3.0",
         "symfony/security": "^2.7 || ^3.0",
         "symfony/validator": "^2.7 || ^3.0",
-        "symfony/finder": "^2.7 || ^3.0"
+        "symfony/finder": "^2.7 || ^3.0",
```

Comment:
We really should enforce sorted dependencies so that we can stop mucking around with where they should "logically" be placed "correctly"... :laughing: